modules/shortcodes/wordads.php

```diff
@@ -53,14 +57,28 @@ static function wordads_shortcode_html( $atts, $content = '' ) {
 			return '<div>' . __( 'The WordAds module is not active', 'jetpack' ) . '</div>';
 		}
 
-		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock">';
-
-		$html .= '</div>';
-
+		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock"></div>';
 		$html = $wordads->insert_inline_ad( $html );
 
 		return $html;
 	}
+
+	public static function gutenblock_render( $attr ) {
+		global $wordads;
+
+		if ( is_feed() || apply_filters( 'wordads_inpost_disable', false ) ) {
```

Comment:
Lets leave a comment on this filter if it has been documented somewhere already. 
if it is a new one. Lets prefix it with `jetpack_`

Reply:
It's been around a while:
https://github.com/Automattic/jetpack/blob/master/modules/wordads/wordads.php#L261

I'll copy the notes over

Reply:
Lets change the copy to instead
/** This filter is already documented in modules/wordads/wordads.php */